Cargo.toml

```
@@ -23,7 +23,7 @@ doc = false
 path = "src/bin/bindgen.rs"
 
 [dependencies]
-clap = "~2.25.0"
+clap = "~2.29.0"
```

Can we update it to the latest maybe? i.e.:
```suggestion
clap = "~2.33.0"
```